scss/tech.scss

```diff
@@ -1,60 +1,59 @@
-@import 'variables/_colors';
+@media only screen and (max-width: 1024px){
```

Comment:
The `media screen` should be placed at the bottom of the file, that way it will override correctly the classes